app/locales/en/translations.js

```diff
@@ -236,6 +236,7 @@ export default {
                     update_comment: 'Update comment',
                 },
                 comment_placeholder: 'Explain the reasoning behind your decision (optional)',
+                comment_length_error: 'Comment exceeded maximum limit of {{limit}} characters by {{difference}}.',
```

Comment:
This seems okay to me, but @laurenbarker, thoughts on this language/approach?

Reply:
The language used for the preprint form in "Abstract is too short (minimum is 20 characters)" so we could do "Comment is too long (maximum is {{limit}})" for consistency.

Reply:
Should it also say how far over it is? If we're letting users type beyond the limit, seems like it'd be good to let them know how much they need to delete.

Reply:
How about "Comment is {{difference}} characters too long (maximum is {{limit}})"?